mpact_api/mpact/views.py

```diff
@@ -26,8 +28,8 @@ class ListenMessages(APIView):
     """
 
     def post(self, request):
-        print(Constants.MESSAGE)
         message = request.data.get(Constants.MESSAGE)
+        logger.debug(message)
```

Comment:
Could we just log when things are **not** what we expect?

There are two issues to keep in mind:
1. This particular project has really bad consequences if sensitive data ends up in the wrong place. This project is for HIV+ young adults. I recognise that, especially in Africa, the intersection between HIV+ and LGBTQ is smaller than in the US and Europe, but in Kenya both LGBTQ and HIV are not only socially stigmatized (hence the use of Telegram), but also being gay is literally illegal, and can result in a prison sentences of up to 14 (very unpleasant) years. We have to take extreme caution with this project's data. I will be following up with the partner to determine the longevity of chat records. If we're going to be storing logs of message content, we'll need to delete those after a fixed amount of time too.
2. We only need debug logs when something goes wrong. So it makes sense to log when we get an exception, and not when we don't.

If we are just using logs to see what's happening when code executes, better not to commit that to the repo. Rather use a debugger like [pdb++](https://pypi.org/project/pdbpp/) and call the webhook with sample payloads. You get all the benefits of the debug logging, and you're also able to explore values as you step through code.

Reply:
Removed loggers in next [PR #3](https://github.com/hussain-mohammed/mpact/pull/3).